libs/coin-modules/coin-celo/src/logic.ts

```diff
@@ -0,0 +1,149 @@
+import {
```

Comment:
Can we rename this utils.ts and move it in a `logic` folder ? To follow Boilerplat @Wozacosta wdt ?